### PR TITLE
Fix instance copy on Linux and macOS :slightly_smiling_face: 

### PR DESCRIPTION
**Pull request by flowln**
damn you .minecraft

### CODE REVIEW

**Comment by luni3359** [comment]
This has been broken?! Do you know to which extent it's broken? I have copied my main instance a few times and I haven't lost my singleplayer worlds.

File: tests/testdata/FileSystem/test_folder/.secret_folder/.secret_file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+oooooo spooky easter egg :oo
```
Comment (Scrumplex) [inline tests/testdata/FileSystem/test_folder/.secret_folder/.secret_file.txt:1]: :trollface:

**Review by Scrumplex — APPROVED** [review]

**Comment by flowln** [comment]
> This has been broken?! Do you know to which extent it's broken? I have copied my main instance a few times and I haven't lost my singleplayer worlds.

Were you using a stable release? If so, you're probably fine, as this seems to be a problem in develop, in which anything inside `.minecraft`, including worlds and mods, weren't being copied over.

**Comment by luni3359** [comment]
Yes I'm on 1.4.2 stable, thanks for letting me know! ❤(ӦｖӦ｡)

**Comment by DioEgizio** [comment]
was actually broken on macOS too, but this pr fixes it anyways

**Pull request closed without merging**